docs/rustdoc/centralized_telescope/telescope/main.md

```diff
@@ -0,0 +1,78 @@
+
+
+---
+> 📖 **See the documentation: [Telescope - Construction with Bounded DFS][crate::docs::centralized]** for further details.
+---
+
+## Creating a Telescope
+
+Constructing an ALBA proof within a **centralized Telescope-bounded DFS setup** requires careful parameter selection to ensure both efficiency and security.
+- The **soundness parameter** ($\lambda_{\text{sec}}$) controls how difficult it is for an adversary to forge a valid proof.
+  - A higher $\lambda_{\text{sec}}$ increases the difficulty of false proofs but may require more computational resources.
+- The **reliability parameter** ($\lambda_{\text{rel}}$) determines the probability that an honest prover can generate a valid proof within the given constraints.
+  - If $\lambda_{\text{rel}}$ is too low, legitimate proof generation may fail, necessitating retries or adjustments in other parameters.
```

Comment:
> necessitating retries or adjustments in other parameters.
The number of retries is set by lambda_rel  so it shouldn't necessicate adjustements there. As you said, it simply implies a higher probability of (proof generation) failure